.github/workflows/integrations.yml

```diff
@@ -0,0 +1,53 @@
+
+name: RMT Integration tests
+
+on: [pull_request]
+
+jobs:
+  test:
+    env:
+      DANGER_GITHUB_API_TOKEN: ${ secrets.GH_TOKEN }
+    services:
+      mysql:
+        image: mysql:latest
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_DATABASE: rmt_test
+          MYSQL_USER: rmt
+          MYSQL_PASSWORD: rmt
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.5.8', '2.6.6', '2.7.2']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true
+        bundler: 1.17
+    - name: Gem cache
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ matrix.ruby-version }}-gems-${{ hashFiles('Gemfile') }}
+        restore-keys: |
+          ${{ matrix.ruby-version }}-gems-
+    - name: Pre-run inline scripts
+      run: |
+        ruby -e "require 'yaml'; puts({'database_test'=>{'host'=>'127.0.0.1','username'=>'rmt','password'=>'rmt','database'=>'rmt_test','adapter'=>'mysql2','encoding'=>'utf8','timeout'=>5000,'pool'=>5}}.to_yaml)" > config/rmt.local.yml
```

Comment:
At first I thought that maybe we needed to have a specific file for tests so to avoid this hack (even though I find it extremely weird that there is no `config/database.yml` file at all.

But then, I believe that this configuration is already identical to what we have in `config/rmt.yml`, at least for the test part. So, why exactly do we need all this? If we really need a `-local` file, can't we simply copy the `rmt.yml` file directly?

Sorry if it's something basic on rmt's code, but all in all looks rather confusing.